### PR TITLE
Add Fuel testnet config and modify Fuel mainnet config

### PR DESCRIPTION
**Pull request by frangrech**

### CODE REVIEW

File: chains/mainnet/fuel-testnet.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "chain_name": "fuel testnet",
+    "registry_name": "fuel testnet",
+    "api": [
+        {"provider": "SimplyStaking", "address": "https://rest-seq.simplystaking.xyz"}
+    ],
+    "rpc": [
+        {"provider": "SimplyStaking", "address": "https://rpc-seq.simplystaking.xyz"}
+    ],
+    "sdk_version": "0.50.5",
```
Comment (dillu24) [inline chains/mainnet/fuel-testnet.json]: for testnet sdk version is `0.50.10` too

File: chains/mainnet/fuel-testnet.json
Comment (dillu24) [inline chains/mainnet/fuel-testnet.json:1]: Does the folder path for the testnet json need to include `mainnet` for it to work?

**Review by dillu24 — CHANGES_REQUESTED** [review]
Reviewed

File: chains/mainnet/fuel-testnet.json
Comment (frangrech) [reply chains/mainnet/fuel-testnet.json:1]: Yes, from https://github.com/ping-pub/explorer/blob/master/chains/README.md
I tried separating and it did not work

> Note: the host name used in the user's browser (displayed in the address bar) to load the explorer app determines which of the chain configuration directories is used. If the host name contains the substring "testnet" (for example: "https://mytestnetwork.example.com/") then chain configuration files in the testnet directory (only) are loaded. Conversely if the host name does not contain that substring then chain configuration files in the mainnet directory (only) are loaded. Remember to bear this behavior in mind when selecting a DNS host name for self hosting the explorer.

Comment (dillu24) [reply chains/mainnet/fuel-testnet.json:1]: ah ok noted.

**Review by dillu24 — APPROVED** [review]

**Pull request merged**